eth/tracers/internal/tracetest/testdata/prestate_tracer/create_create.json

```diff
@@ -11,8 +11,7 @@
     "number": "1",
     "stateRoot": "0xd2ebe0a7f3572ffe3e5b4c78147376d3fca767f236e4dd23f9151acfec7cb0d1",
     "timestamp": "1699617692",
-    "withdrawals": [],
-    "withdrawalsRoot": "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421",
+    "totalDifficulty": "0",
```

Comment:
Total difficulty is removed now from API. I think this line (and in the diffs above) can be removed.